README.md

```diff
@@ -43,6 +43,7 @@ components as indicated by the Maturity table below.
 | metrics/\*<br>collector/metrics/* | Stable | [Stable](docs/specification.md#json-protobuf-encoding) |
 | trace/\*<br>collector/trace/* | Stable | [Stable](docs/specification.md#json-protobuf-encoding) |
 | logs/\*<br>collector/logs/* | Stable | [Stable](docs/specification.md#json-protobuf-encoding) |
+| profiles/\*<br>collector/profiles/* | Experimental | [Experimental](docs/specification.md#json-protobuf-encoding) |
```

Comment:
The `maturity-matrix.yaml` does not list `experimental` for `apiMaturityLevels`. Would `alpha` be applicable?

Reply:
We can update that YAML.  `experimental` matches the terminology we use elsewhere in the spec and we have not added anything to the Proto since going stable, where `experimental` would be needed.

Reply:
Now that [OTEP0232](https://github.com/open-telemetry/oteps/blob/main/text/0232-maturity-of-otel.md#maturity-levels) is accepted we need to apply it to Otel including this repo. Not in this PR though, a separate one.